testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -0,0 +1,337 @@
+    /// <exception cref="Exception"></exception>
+    public static Func<bool> ResultIsSet(bool useTimeoutException = true)
+    {
+        var startWaitTime = Time.time;
+        return () =>
+        {
+            if (Time.time - startWaitTime > maxWaitTimeout)
+            {
+                if (useTimeoutException)
+                {
+                    throw new Exception($"timeout while waiting for results, didn't get results for {Time.time - startWaitTime} seconds");
+                }
+
+                return true;
+            }
+
+            foreach (var clientIdAndTestResultList in Instance.m_TestResultsLocal)
+            {
+                if (clientIdAndTestResultList.Value.Count > 0)
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        };
+    }
+
+    public static Func<bool> ConsumeClientIsFinished(ulong clientId, bool useTimeoutException = true)
+    {
+        var startWaitTime = Time.time;
+        return () =>
+        {
+            if (Time.time - startWaitTime > maxWaitTimeout)
+            {
+                if (useTimeoutException)
+                {
+                    throw new Exception($"timeout while waiting for client finished, didn't get results for {Time.time - startWaitTime} seconds");
+                }
+                else
+                {
+                    return true;
+                }
+            }
+
+            if (Instance.m_ClientIsFinished.ContainsKey(clientId) && Instance.m_ClientIsFinished[clientId])
+            {
+                Instance.m_ClientIsFinished[clientId] = false; // consume
+                return true;
+            }
+
+            return false;
+        };
+    }
+
+    [ServerRpc(RequireOwnership = false)]
+    public void ClientFinishedServerRpc(ServerRpcParams p = default)
+    {
+        // signal from clients to the server to say the client is done with it's task
+        m_ClientIsFinished[p.Receive.SenderClientId] = true;
+    }
+
+    public void InvokeFromMethodActionRpc(Action<byte[]> methodInfo, params byte[] args)
+    {
+        var methodInfoString = GetMethodInfo(methodInfo);
+        InvokeFromMethodNameClientRpc(methodInfoString, args, new ClientRpcParams { Send = new ClientRpcSendParams { TargetClientIds = AllClientIdExceptMine.ToArray() } });
+    }
+
+    public void InvokeFromMethodActionRpc(Action methodInfo)
+    {
+        var methodInfoString = GetMethodInfo(methodInfo);
+        InvokeFromMethodNameClientRpc(methodInfoString, null, new ClientRpcParams { Send = new ClientRpcSendParams { TargetClientIds = AllClientIdExceptMine.ToArray() } });
+    }
+
+    [ClientRpc]
+    public void TriggerActionIDClientRpc(string actionID, byte[] args, ClientRpcParams clientRpcParams = default)
+    {
+        Debug.Log($"received RPC from server, client side triggering action ID {actionID}");
+        try
+        {
+            // ExecuteStepInContext.allActions[actionID].Invoke(args);
```

Comment:
Oh?

Reply:
same as the other commented out line, this is uncommented in next PR here https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/924/files#diff-b8faa87987f742aa41389349b69aa57ae459a1446feaf0a6c3751b22fe068d9bR259
(I could have just removed and readded it in next PR, sorry for the confusion)